L1Trigger/L1THGCal/python/hgcalTriggerPrimitiveDigiProducer_cfi.py

```diff
@@ -65,7 +65,11 @@
 
 C3d_parValues = cms.PSet( dR_multicluster = cms.double(0.01), # dR in normalized plane used to clusterize C2d
                           minPt_multicluster = cms.double(0.5), # minimum pt of the multicluster (GeV)
-                          calibSF_multicluster = cms.double(1.084)
+                          calibSF_multicluster = cms.double(1.084),
+                          type_multicluster = cms.string('DBSCANC3d'), # 'dRC3d' for the cone algorithm 
```

Comment:
This algo should not be used as default for the moment since no performance studies have been shown yet.